src/webui/www/private/views/preferences.html

```diff
@@ -3061,6 +3070,12 @@
             settings["announce_to_all_trackers"] = $("announceAllTrackers").checked;
             settings["announce_to_all_tiers"] = $("announceAllTiers").checked;
             settings["announce_ip"] = $("announceIP").value;
+            const announce_port = Number($("announcePort").value);
+            if (isNaN(announce_port) || (announce_port < 0) || (announce_port > 65535)) {
```

Comment:
MDN suggest using `Number.isNaN()` over `isNaN()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN
```suggestion
            if (Number.isNaN(announce_port) || (announce_port < 0) || (announce_port > 65535)) {
```

Reply:
I just saw other code were using isNaN(), so you can ignore this suggestion. I'll change all of them in a later PR.

Reply:
Applied the suggested change to make sure you can fix the others without waiting on this PR.